<launch>
    <param name="robot_description" textfile="$(find first_urdf)/urdf/gazebonatak.urdf"/>
    <include file="$(find gazebo_ros)/launch/empty_world.launch" />
    <node name="spawn_urdf" pkg="gazebo_ros" type="spawn_model" args="-param robot_description -urdf -model 2cyl" />
    <rosparam file="$(find first_urdf)/config/controller.yaml"/>
    <node name= "controller_spawner" pkg= "controller_manager" type="spawner" args="arm_controller" />
    <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher"/>

</launch>
